src/doc/src/reference/rust-version.md

```diff
@@ -24,6 +24,9 @@ Cargo will provide clearer diagnostics about the insufficient toolchain version
 This affects all [Cargo targets](cargo-targets.md) in the package, including binaries, examples, test suites,
 benchmarks, etc.
 
+Cargo will not build your package on an unsupported toolchain, unless the `--ignore-rust-version` option
+below is specified.
```

Comment:
The previous paragraph is attempting to say the same thing.  By adding this, we are making it come across as these are saying two different things which can make this more confusing.

I can see being more upfront that we will always error with the intent of providing a clearer diagnostic than what you could get otherwise and having a comment at the end that in case the dependency happens to work, you can opt-in to an unsupported build of the package with `--ignore-rust-version`.

Reply:
That's fair! I'll edit it to be part of the previous paragraph like you suggested.

Reply:
Pushed new commit with suggestions applied. Thank you!